Dockerfile-superbuild

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:18.04
```

Comment:
Can you move it to `superbuild/Dockerfile`? `Dockerfile-superbuild` is not recognized as a standard name for Docker. Similarly, you can move the initial Docker file to `ci/Dockerfile`.